driver/src/main/com/mongodb/DBCollection.java

```diff
@@ -286,9 +285,9 @@ public WriteResult insert(final List<? extends DBObject> documents, final WriteC
     /**
      * <p>Insert documents into a collection. If the collection does not exists on the server, then it will be created. If the new document
      * does not contain an '_id' field, it will be added.</p>
-     * 
-     * <p>If the value of the continueOnError property of the given {@code InsertOptions} is true, 
-     * that value will override the value of the continueOnError property of the given {@code WriteConcern}. Otherwise, 
+     *
```

Comment:
Its difficult as I have remove whitespace on save - which should be the default, at least has been ever since I got the updated intellij configs.

I could turn it off and then manually redo this whole pull request - but I'm not sure its worth it? - as we shouldn't have trailing whitespace.